src/main/java/edu/project2/GenerateMaze.java

```diff
@@ -0,0 +1,185 @@
+package edu.project2;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Deque;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.Random;
+
+public class GenerateMaze {
```

Comment:
Названия классов принято делать существительными
```suggestion
public class MazeGenerator {
```